src/solvers/dgsem_tree/dg_2d_parabolic.jl

```diff
@@ -513,11 +515,11 @@ function calc_boundary_flux_by_direction_divergence!(surface_flux_values::Abstra
     return nothing
 end
 
-function prolong2mortars!(cache, flux_viscous::Tuple{AbstractArray, AbstractArray},
+function prolong2mortars!(cache, flux_viscous::Vector{Array{uEltype, 4}},
```

Comment:
I can't comment on it, but `prolong2mortars` is called with `cache` and not `cache_parabolic` right? If so, please add some comments explaining why this is (I believe it's because `cache_parabolic` is missing mortars)

Reply:
Yes exactly, all mortar business is handled by the (hyperbolic) `cache`.

Reply:
Yes, definitively something to discuss. Maybe it is also worth reconsidering whether we need the interfaces in cache_parabolic.

Reply:
It would indeed be nice to remove the interfaces from `cache_parabolic` for a cleaner implementation. If we don't put mortars there, it would be more consistent to not have interfaces there either.

Reply:
From today's meeting: 

1. we'll first merge [this PR on AMR for parabolic terms on `TreeMesh`](https://github.com/trixi-framework/Trixi.jl/pull/1629)
2. next, we'll work on merging the [PR for AMR with parabolic terms on `P4estMesh`](https://github.com/trixi-framework/Trixi.jl/pull/1662)
3. we'll then make a PR for `TreeMesh` to see if removing `interfaces` from `cache_parabolic` simplifies the implementation. 
4. if the PR in step 3 goes through, we'll make a similar PR for `P4estMesh`